File: app/src/main/java/ua/edu/deanoffice/mobile/studentchdtu/MainActivity.java
package ua.edu.deanoffice.mobile.studentchdtu;

import android.content.Intent;
import android.os.Bundle;


import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.util.Log;
import android.view.View;
import android.widget.ArrayAdapter;
import android.widget.Button;
import android.widget.Spinner;

import java.util.ArrayList;

import ua.edu.deanoffice.mobile.studentchdtu.service.pojo.ApplicationTypePOJO;
import ua.edu.deanoffice.mobile.studentchdtu.service.pojo.RenewApplicationData;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button = findViewById(R.id.button);
        ArrayList<ApplicationTypePOJO> applicationTypeList = new ArrayList<ApplicationTypePOJO>();

        final Spinner spinner = findViewById(R.id.spinner);
        String[] temp = new String[]{
                "Перескладання іспиту для підвищення оцінки з метою отримання диплому з відзнакою",
                "Поновлення до складу здобувачів вищої освіти після академічної відпустки",
                "Відрахування за власним бажанням"
        };

        ArrayAdapter adapter = new ArrayAdapter(this, R.layout.multiline_spinner_dropdown_item, temp);

        spinner.setAdapter(adapter);


        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, RetakeApplicationActivity.class);

                switch (1+(int)spinner.getSelectedItemId()){
                    case 1:
                        intent = new Intent(MainActivity.this, RetakeApplicationActivity.class);
                        break;
                    case 2:
                        intent = new Intent(MainActivity.this, RenewApplicationActivity.class);
                        break;
                    case 3:
                        intent = new Intent(MainActivity.this, DeducApplicationActivity.class);
                        break;

                        default:
                            break;
                }
                Log.d("Test", "enter");
                MainActivity.this.startActivity(intent);
                finish();
            }
        });
    }



}

File: app/src/main/java/ua/edu/chdtu/deanoffice/mobile/myapplication/MainActivity.kt
package ua.edu.chdtu.deanoffice.mobile.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ArrayAdapter
import android.widget.Spinner
import ua.edu.chdtu.deanoffice.mobile.service.POJO.ApplicationTypePOJO


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val button = findViewById<Button>(R.id.button)
        
        var applicationTypeList = ArrayList<ApplicationTypePOJO>()

        //applicationTypeList = Utils.JSONtoArrayObjects(Client.getApplicationList().response!!.body)
        //Client.getApplication(0, Utils.retakeApplicationDataToJSON(RetakeApplicationData("Mathematics", 2)) )
        //println(Utils.retakeApplicationDataToJSON(RetakeApplicationData("Mathematics", 2)))

        val spinner = findViewById(R.id.spinner) as Spinner
        var temp = arrayListOf<String>(
                "Перескладання іспиту для підвищення оцінки з метою отримання диплому з відзнакою",
                "Поновлення до складу здобувачів вищої освіти після академічної відпустки",
                "Відрахування за власним бажанням")
        val adapter = ArrayAdapter(this, R.layout.multiline_spinner_dropdown_item, temp.toArray())

        spinner.adapter = adapter

        button.setOnClickListener {
            var intent = Intent(this, RetakeApplicationActivity::class.java)

            when(1+spinner.selectedItemId){
                1L -> { intent = Intent(this, RetakeApplicationActivity::class.java) }
                2L -> { intent = Intent(this, RenewApplicationActivity::class.java) }
                3L -> { intent = Intent(this, DeducApplicationActivity::class.java) }
            }

            startActivity(intent)
        }



    }
}


